react-native/ios/Keybase/Images.xcassets/service_coinbase.imageset/Contents.json

```diff
@@ -1,21 +0,0 @@
-{
```

Comment:
All images are now in the js land and not in the platform specific project folders